extensions/aria_extension_tosca/simple_v1_0/types.py

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from aria.utils.collections import FrozenDict, FrozenList
+from aria.utils.collections import (FrozenDict, FrozenList)
```

Comment:
if by adding the module prefix makes the code unreadable, maybe we need to refactor the structure?